kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/ChecksumSimpleComparisonSuite.scala

```diff
@@ -46,10 +47,14 @@ import org.apache.spark.sql.functions.col
  * This suite ensures that both implementations generate consistent checksums
  * for various table operations.
  */
-class ChecksumSimpleComparisonSuite extends DeltaTableWriteSuiteBase with TestUtils {
+trait ChecksumComparisonSuiteBase extends DeltaTableWriteSuiteBase with TestUtils {
 
   private val PARTITION_COLUMN = "part"
 
+  protected def getPostCommitHookType: PostCommitHookType
+
+  protected def maybeDeleteChecksum(tablePath: String, version: Long): Unit = {}
```

Comment:
can you document when to call this? and why we have it?

Reply:
With the suggestion of do it before commit, this method is not needed